<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridArrayQt</class>
 <widget class="QMainWindow" name="GridArrayQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GridArrayQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>241</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupFileInfo">
         <property name="title">
          <string>FileInfo</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_GridH"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_GridW">
              <property name="text">
               <string>Grid width :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_GridW"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_GridColor">
              <property name="text">
               <string>Grid color :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_GridH">
              <property name="text">
               <string>Grid height :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="hSlider_GridColor">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupLoadPic">
         <property name="title">
          <string>Load</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QSlider" name="hSlider_Gray">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radio_Gray">
              <property name="text">
               <string>Gray :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="hSlider_Red">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radio_Red">
              <property name="text">
               <string>Red :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="hSlider_Blue">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radio_Blue">
              <property name="text">
               <string>Blue :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="hSlider_Green">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radio_Green">
              <property name="text">
               <string>Green :</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="btn_LoadAr">
                <property name="text">
                 <string>Load Array</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_LoadPic">
                <property name="text">
                 <string>Load Pic</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_UnloadImg">
                <property name="text">
                 <string>Unload Pic</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupText">
         <property name="title">
          <string>Text</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_FontType">
              <property name="text">
               <string>Font :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_FontSize">
              <property name="text">
               <string>Size :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QFontComboBox" name="fontComboBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btn_Accept">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="btn_Bold">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Italic">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Underline">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockW_Ctrl">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_GridW</tabstop>
  <tabstop>lineEdit_GridH</tabstop>
  <tabstop>hSlider_GridColor</tabstop>
  <tabstop>radio_Gray</tabstop>
  <tabstop>hSlider_Gray</tabstop>
  <tabstop>radio_Red</tabstop>
  <tabstop>hSlider_Red</tabstop>
  <tabstop>radio_Green</tabstop>
  <tabstop>hSlider_Green</tabstop>
  <tabstop>radio_Blue</tabstop>
  <tabstop>hSlider_Blue</tabstop>
  <tabstop>btn_LoadPic</tabstop>
  <tabstop>btn_UnloadImg</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>btn_Bold</tabstop>
  <tabstop>btn_Italic</tabstop>
  <tabstop>btn_Underline</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
